nodejs/week1/mealSharing/src/backend/routes/cheap_meals.js

```diff
@@ -0,0 +1,16 @@
+const cheapMeals = function (req, res) {
+    const jsonMealsObj = require('../data/meals.json'); 
+    const jsonReviewMeal = require('../data/reviews.json');    
+    const cheapMealArr = jsonMealsObj.filter((item) =>  {
+        return item.price < 80;
+      });
+    const cheapReviewArr = jsonMealsObj.map((el) => {
+          return el.review = jsonReviewMeal.filter((item) => {
+           if (el.id === item.mealID) {
+              return item.mealID === el.id
```

Comment:
Line 9 and 10 are a bit odd. At first, you are checking if `el.id === item.mealID`, after you return the result of the same operation. Instead, you want to just filter them based on this result. Can you figure a way of writing it simpler? The same pattern repeats in other files as well.